meetings/2021-07/july-15.md

```diff
@@ -0,0 +1,734 @@
+
+WH: I'm trying to understand your slide. The `fibonacciSequence` function returns an array from which Realm?
+
+CP: It is an array-like, and let us assume that it is from the realm that is executing this code on, which is not the other realm. Let's assume that for the sake of simplicity.
+
+JHD: I mean, the specific potential issue here is that this will return an array whose `.constructor` is `otherRealmGlobalThis.Array`. That's not `Array`. But this is only an issue If somebody's using instance of on arrays, which we've all been trained not to do for 10 years, 
```

Comment:
```suggestion
JHD: I mean, the specific potential issue here is that this will return an array whose `.constructor` is `otherRealmGlobalThis.Array`. That's not `Array`. But this is only an issue If somebody's using `instanceof` on arrays, which we've all been trained not to do for 10 years.
```